### DISCUSSION TITLE
Wiki page with step-by-step guidelines?

### DISCUSSION THREAD

**Discussion by e-t-l** [AveYo/MediaCreationTool.bat#25] [Ideas]
I'm fairly tech-savvy, and I'm finding the ReadMe a little difficult to parse. Would anyone familiar with the program like to write out a short step-by-step page for the repo wiki so the less-tech-literate can understand what exactly they need to do? (e.g. #7 made it clear that many folks don't understand what exactly they need to do to upgrade their computer to Win11 with TPM bypasses. ...and to be quite frank, I might be one of them.)

#15 is a good step in the right direction, but it could easily be made more accessible than saying, "I think it's somewhat self-explanatory."

**Comment by e-t-l** [comment]
The following steps from @Ehijoshua at [verifiedngnews](https://www.verifiedngnews.com/2-methods-on-how-to-bypass-windows-11s-tpm-requirements/) is also helpful:

> Step 1. First of all, open your web browser and navigate to [this Github page](https://gist.github.com/AveYo/c74dc774a8fb81a332b5d65613187b15#file-mediacreationtool-zip).
> 
> Step 2. Now scroll down and download the MediaCreationTool.zip file.
> 
> Step 3. Now extract the zip file that you have downloaded and copy the MediaCreationTool.bat on a safe folder.
> 
> Step 4. Now you need to run the MediaCreationTool.bat file on your system.
> 
> Step 5. Now you will be asked to select the MCT Version. Here you need to select 11 as the MCT version.
> 
> Step 6. In the next window, you will be asked to select either Create USB or Create ISO. If you wish to create a bootable USB device, click on Create USB option.
> 
> Step 7. Now you will see the Windows 10 setup wizard. Here you need to select the USB Flash drive option.
> 
> Step 8. On the next Window, select the Removable drive and click on the Next button.
> 
> Step 9. Now the Media Creation tool will download Windows 11 from Microsoft’s servers. It will say ‘Downloading Windows 10’, but it will be downloading Windows 11.

However, it still leaves some ambiguity. For example, in order to install Win 11 on the local system with bypassed TPM, is burning a removable drive the only option that works?

**Comment by AveYo** [comment]
The first 4 options (Auto Upgrade, Create ISO, Create USB, Select) will add automatic bypass when booting from media.
Under windows, just use the added `auto.cmd` in the media to launch setup with TPM bypass.

Only the last option (Default MCT) will create a vanilla, unmodified media without TPM bypass.
